package.json

```diff
@@ -8,7 +8,7 @@
     "precommit": "jscs . --fix && eslint ./client/app && npm test",
     "postinstall": "npm run build",
     "dev-client": "gulp",
-    "dev-server": "env $(cat .env) nodemon server/index.js --exec babel-node",
+    "dev-server": "nf run nodemon server/index.js",
```

Comment:
The current Procfile already has the correct `dev` command in there. I'll switch it over to using that.